src/map/Map.js

```diff
@@ -1108,7 +1108,7 @@ export const Map = Evented.extend({
 
 		container.classList.add(...classes);
 
-		const position = DomUtil.getStyle(container, 'position');
+		const {position} = container.style;

```

Comment:
Why not use computed style here? Container position could be defined in CSS, and then it would be overriden with `relative`, right?

Reply:
Yeah, you are totally right. We should be using computed styles for all of these.